gitea/README.md

```diff
@@ -34,4 +37,73 @@ cat <<EOF | kubectl apply -f -
     username: ...      
     password: ...
 EOF
+```
+
+# OpenShift
```

Comment:
Hmm. This is OK for now. But it gets at a general problem we haven't solved yet and we are likely to see over and over again.

Essentially, deploying this package into a particular environment creates the need to apply some additional resources. This could be treated as a separate, dependent package.

We should think about is whether we should have some sort of "deployment context" resource that our different packages understand, or a an "environment". This could be done as a binding of sorts, like described in https://github.com/GoogleContainerTools/kpt/issues/3973. If you bind the "openshift" resource to it, it would pull these in as a package dependency or perhaps as some sort of "mix in" concept we haven't yet figured out.

Reply:
I think an alternative before we figure this out is to just use a derivative package.

We can clone the current Gitea package into a new GCP Gitea package and an OpenShift Gitea package, and add the appropriate resources.